mtgjson4/mtg_builder.py

```diff
@@ -333,6 +348,14 @@ def determine_layout_and_div_name(soup: bs4.BeautifulSoup,
         if c_original_text:
             card_info['originalText'] = c_original_text
 
+        return card_info
+
+    @staticmethod
+    def combine_all_parts_into_a_card(main_parts, original_parts, legal_parts, foreign_parts) -> Dict[str, Any]:
+        dict_builder1 = {**main_parts, **original_parts}
+        dict_builder2 = {**legal_parts, **foreign_parts}
+        return {**dict_builder1, **dict_builder2}
```

Comment:
why not `return {**main_parts, **original_parts, **legal_parts, **foreign_parts}`?

You should also `-> CardDescription` on this method so you can push the return type up the stack from there

Reply:
Was based on SO and didn't know if I could do that, thanks for the tip